.circleci/config.yml

```diff
@@ -352,6 +358,43 @@ jobs:
             RELEASE_CHANNEL: experimental
           command: yarn test-build-prod --maxWorkers=2
 
+  test_ms_windows:
+    executor:
+      name: win/default
+    environment: *ms_windows_environment
+    steps:
+      - run:
+          # Fix line endings in Windows.
+          command: git config --global core.autocrlf input
+      - checkout
+      - restore_cache:
+          keys:
+            - v2-win-node-{{ arch }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - v2-win-node-{{ arch }}-{{ .Branch }}-
+            - v2-win-node-{{ arch }}-
+      - run:
+          command: nvm install 10.18.1
```

Comment:
Looks like 12.x is used on Linux, any reason to use 10.x? (it'll be in maintenance soon)

Reply:
10.x is what the AppVeyor config uses, not sure if we should try upgrading?